src/librustc_typeck/region_to_region.rs

```diff
@@ -0,0 +1,9 @@
+use crate::astconv::AstConv;
+
+use crate::middle::resolve_lifetime as rl;
+use rustc_hir as hir;
+use rustc_middle::ty;
+
+pub trait RegionToRegion<'tcx>: AstConv<'tcx> {}
```

Comment:
What's this for?

Reply:
Oops, it's a remnant of an earlier test to split up the module. It shouldn't exist